testdata/go/v3/memcached-operator/docs/monitoring/metrics.md

```diff
@@ -0,0 +1,11 @@
+# Memcached Operator Metrics
+This document aims to help users that are not familiar with metrics exposed by the memcached-operator.
+All metrics documented here are auto-generated by the utility tool `monitoring/metricsdocs` and reflects exactly what is being exposed.
+
+## Memcached Operator Metrics List
+### memcached_deployment_size_undesired_count_total
+Total number of times the deployment size was not as desired. Type: Counter.
+## Developing new metrics
```

Comment:
@avlitman I think we should add a new line before this line

Reply:
Shirly did you take a look at the real file: https://github.com/operator-framework/operator-sdk/blob/c1dea99dffa06c59fdd7c03871696f9cfcf9c5a6/testdata/go/v3/memcached-operator/docs/monitoring/metrics.md
?